rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/message_introspection.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014-2015 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROSIDL_TYPESUPPORT_INTROSPECTION_C__MESSAGE_INTROSPECTION_H_
+#define ROSIDL_TYPESUPPORT_INTROSPECTION_C__MESSAGE_INTROSPECTION_H_
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "rosidl_generator_c/message_type_support.h"
+
+#include "rosidl_typesupport_introspection_c/visibility_control.h"
+
+typedef struct
+// ROSIDL_TYPESUPPORT_INTROSPECTION_C_PUBLIC
```

Comment:
Yeah, I removed them for now. We can put them in when we need them down the road.